sass/includes/_header.scss

```diff
@@ -243,7 +243,14 @@
 
   &__overlay-input {
     // using opacity instead of display: none allows the input to be toggled by keyboards
-    opacity: 0;
+    //opacity: 0;
```

Comment:
@JohnHeeryTNA after doing a bit more testing locally I've realised this change isn't needed, I'd misunderstood what axe was complaining about.

We actually just need to remove the `tabindex="-1"` attribute from the input instead - this will mean the menu toggle button is correctly focusable on keyboards.

Sorry for the confusion! A much more straightforward fix than anticipated - we can remove this styling update.